velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -688,8 +690,81 @@ TpchPlan TpchQueryBuilder::getQ18Plan() const {
   return context;
 }
 
+TpchPlan TpchQueryBuilder::getQ19Plan() const {
+  static const std::string kLineitem = "lineitem";
+  static const std::string kPart = "part";
+  std::vector<std::string> lineitemColumns = {
+      "l_partkey",
+      "l_extendedprice",
+      "l_discount",
+      "l_shipinstruct",
+      "l_shipmode",
+      "l_quantity"};
+  std::vector<std::string> partColumns = {
+      "p_partkey", "p_brand", "p_container", "p_size"};
+
+  auto lineitemSelectedRowType = getRowType(kLineitem, lineitemColumns);
+  const auto& lineitemFileColumns = getFileColumnNames(kLineitem);
+  auto partSelectedRowType = getRowType(kPart, partColumns);
+  const auto& partFileColumns = getFileColumnNames(kPart);
+
+  auto planNodeIdGenerator = std::make_shared<PlanNodeIdGenerator>();
+  core::PlanNodeId lineitemScanNodeId;
+  core::PlanNodeId partScanNodeId;
+
+  std::string filterExpr =
+      " ((p_brand = 'Brand#12')"
+      " AND (l_quantity between 1.0 and 11.0)"
+      " AND ((p_container = 'SM CASE') OR (p_container = 'SM BOX') OR (p_container = 'SM PACK') OR (p_container = 'SM PKG'))"
+      " AND (p_size BETWEEN 1 AND 5))"
+      " OR "
+      " ((p_brand ='Brand#23')"
```

Comment:
add some indentation to the OR clause predicates here and below?

Reply:
I meant something like this
```
        p_partkey = l_partkey
    AND p_brand = 'Brand#12'
    AND p_container IN ('SM CASE', 'SM BOX', 'SM PACK', 'SM PKG')
    AND l_quantity >= 1
    AND l_quantity <= 1 + 10
    AND p_size BETWEEN 1 AND 5
    AND l_shipmode IN ('AIR', 'AIR REG')
    AND l_shipinstruct = 'DELIVER IN PERSON')
    OR (p_partkey = l_partkey
        AND p_brand = 'Brand#23'
        AND p_container IN ('MED BAG', 'MED BOX', 'MED PKG', 'MED PACK')
        AND l_quantity >= 10
        AND l_quantity <= 10 + 10
        AND p_size BETWEEN 1 AND 10
        AND l_shipmode IN ('AIR', 'AIR REG')
        AND l_shipinstruct = 'DELIVER IN PERSON')
    OR (p_partkey = l_partkey
        AND p_brand = 'Brand#34'
        AND p_container IN ('LG CASE', 'LG BOX', 'LG PACK', 'LG PKG')
        AND l_quantity >= 20
        AND l_quantity <= 20 + 10
        AND p_size BETWEEN 1 AND 15
        AND l_shipmode IN ('AIR', 'AIR REG')
        AND l_shipinstruct = 'DELIVER IN PERSON');

```

Reply:
Oh ok, thanks for sharing, will change accordingly